tests/test_random_state.py

```diff
@@ -35,6 +35,9 @@ def get_default_floating():
 
 
 class TestNormal:
+    # TODO: Temporary skip due to incorrect results in public CI
+    # (ARM architecture) with the new MKL package 2024.2.0
```

Comment:
please create JIRA case or GH issue and leave the number here as a reference

Reply:
done